sdks/python/apache_beam/runners/dataflow/internal/apiclient_test.py

```diff
@@ -329,7 +402,7 @@ def test_harness_override_default_in_released_sdks(self):
     env = apiclient.Environment([], #packages
                                 pipeline_options,
                                 '2.0.0', #any environment version
-                                FAKE_PIPELINE_URL)
+                                FAKE_PIPELINE_URL, None, None)
```

Comment:
Can None be an optional argument rather than passing it everywhere here?

Reply:
Done.